go.mod

```diff
@@ -665,6 +665,7 @@ require (
 	github.com/gocomply/scap v0.1.2-0.20230531064509-55a00f73e8d6
 	github.com/godror/godror v0.37.0
 	github.com/jmoiron/sqlx v1.3.5
+	github.com/ken-schneider/dublin-traceroute v0.0.3-0.20240221203624-d86186ee6723
```

Comment:
Shall we use a fork based on `github.com/DataDog` instead ?

I think it's better to not depend on personal GH account if we want to merge the PR to `main`.